src/screens/Home/Home.tsx

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
u r such a good developer please don't leave... :( you're SOO GOOD!!!

Reply:
:'''''''''''' ahhhhhhhhhh